main/lsp/requests/hover.cc

```diff
@@ -216,15 +156,20 @@ unique_ptr<ResponseMessage> HoverTask::runRequest(LSPTypecheckerDelegate &typech
             retType = core::Types::untypedUntracked();
         }
         typeString = retType.showWithMoreInfo(gs);
-    } else if (auto l = resp->isLiteral()) {
-        auto [send, kwargName] = enclosingSendForKwarg(gs, *l, queryResponses);
-        if (send != nullptr) {
-            typeString = handleHoverKeywordArg(gs, send, kwargName);
-        }
+    } else if (auto *kw = resp->isKeywordArg()) {
+        // Have do do this one separate, because it was stolen out of the queryResponses vector
```

Comment:
```suggestion
        // Have to do this one separately, because it was stolen out of the queryResponses vector
```

Reply:
will land with the stacked change, thanks

Reply:
https://github.com/sorbet/sorbet/pull/9428/commits/790b875f9f4b306263940a681bf2bea0821c9d8f